server/StudySharp.ApplicationServices/Commands/RemoveTagByIdCommand.cs

```diff
@@ -13,18 +15,30 @@ public sealed class RemoveTagByIdCommand : IRequest<OperationResult>
         public int Id { get; set; }
     }
 
+    public class RemoveTagByIdCommandValidator : AbstractValidator<RemoveTagByIdCommand>
+    {
+        public RemoveTagByIdCommandValidator(ITagRules rules)
+        {
+            RuleFor(_ => _.Id)
+                .NotEmpty()
+                .WithMessage(string.Format(ErrorConstants.FieldIsRequired, nameof(Tag.Id)))
+                .MustAsync((_, token) => rules.IsTagIdExistAsync(_))
```

Comment:
cancellationToken

Reply:
Do we need it here? 
If I'm not mistaken, if we will have problems with IsTagIdExistAsync, MustAsync end this operation by his token

Reply:
Take a look at the example: https://github.com/Ebashers/StudySharp/blob/dev/server/StudySharp.ApplicationServices/ValidationRules/Auth/LoginRules.cs

I just did the same what we've done with the handlers: mediator provides us cancellation token that we are using in all async operations. Fluent validator does the same. So why don't we use it here like we do this in mediator?

Reply:
Make sense, ok